server/v2/stf/core_event_service.go

```diff
@@ -55,14 +58,14 @@ func (em *eventManager) EmitNonConsensus(event gogoproto.Message) error {
 // TypedEventToEvent takes typed event and converts to Event object
 func TypedEventToEvent(tev gogoproto.Message) (event.Event, error) {
 	evtType := gogoproto.MessageName(tev)
-	evtJSON, err := gogoproto.Marshal(tev)
-	if err != nil {
+	buf := new(bytes.Buffer)
+	jm := &jsonpb.Marshaler{OrigName: true, EmitDefaults: true, AnyResolver: nil}
```

Comment:
I am using jsonpb now but without the `types.UnpackInterfaces` to not add codec types.